tasks.js

```diff
@@ -60,17 +60,71 @@ console.log('Is average correct?: ', avg.toFixed(2) === '68333.33')
 //true
   
   /* Task - Which department is the biggest? */
-const biggestDept: string = '' // write your function here
-  
-console.log('Is biggest department correct?: ', biggestDept === 'Marketing')
-  
+// const biggestDept: string = '' // write your function here
+ 
+// const marketing = employees.filter(employee => employee.department === "Marketing").length;
+// const sales = employees.filter(employee => employee.department === "Sales").length;
+// const it = employees.filter(employee => employee.department === "IT").length;
+
+// const biggestDept = marketing >= sales && marketing >= it ? "Marketing" : sales >= it ? "Sales" : "IT";
+
+// console.log('Is biggest department correct?: ', biggestDept === 'Marketing')
+ 
+
+
+const departmentCounts = {};
+
+employees.forEach(employee => {
+  const dept = employee.department;
```

Comment:
please start using destructuring :)
```suggestion
  const { department } = employee;
```